notebooks/notebook_template.ipynb

```diff
@@ -255,7 +255,7 @@
       },
```

Comment:
<p>Redirect to detailed doc regarding service account permissions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>